ui/src/app/shared/components/application-selector/application-selector.tsx

```diff
@@ -5,13 +5,21 @@ import * as models from '../../models';
 import {appInstanceName, appQualifiedName, ComparisonStatusIcon, HealthStatusIcon, OperationPhaseIcon} from '../../../applications/components/utils';
 import {AuthSettingsCtx} from '../../context';
 
-export const ApplicationSelector = ({apps, formApi}: {apps: models.Application[]; formApi: FormFunctionProps}) => {
+export const ApplicationSelector = ({apps, formApi, terminate}: {apps: models.Application[]; formApi: FormFunctionProps; terminate?: boolean}) => {
     const useAuthSettingsCtx = React.useContext(AuthSettingsCtx);
     return (
         <>
             <label>
                 Apps (<a onClick={() => apps.forEach((_, i) => formApi.setValue('app/' + i, true))}>all</a>/
```

Comment:
I know this is a shared component but is it more intuitive to just show only the apps that are in the process of being synced? eg. those that can be terminated

Reply:
Yes, it now only shows the apps that are in the process of being synced in the terminate panel.